examples/archive.html

```diff
@@ -91,39 +161,63 @@ <h3 id="offcanvasRightLabel">Images</h3>
 
     setupMap();
     setupCollection();
-    
+
     form.addEventListener('submit', (event) => {
       event.preventDefault();
       const url = input.value.replace('details', 'metadata');
-      axios.get(url)
+      let fetchedFrom;
+      axios
+        .get(url)
         .then((response) => {
           if (response.data.files && response.data.files.length != 0) {
             let imageCount = 0;
-            response.data.files.forEach(file => {
-              if (file.format === 'PNG' || file.format === 'JPEG'){
+            response.data.files.forEach((file) => {
+              if (file.format === 'PNG' || file.format === 'JPEG') {
                 let imageRow = document.createElement('div');
                 let image = new Image(150, 150);
                 let placeButton = document.createElement('a');
-
-                placeButton.classList.add('btn', 'btn-sm', 'btn-outline-secondary', 'place-button');
+                fetchedFrom = document.createElement('p');
+                fetchedFromUrl = document.createElement('a');
+                fetchedFromUrl.setAttribute('href', input.value);
+                fetchedFromUrl.setAttribute('target', '_blank');
+                fetchedFromUrl.innerHTML = 'here';
+                fetchedFrom.appendChild(fetchedFromUrl);
+
+                placeButton.classList.add(
+                  'btn',
+                  'btn-sm',
+                  'btn-outline-secondary',
+                  'place-button'
+                );
                 placeButton.innerHTML = 'Place on map';
 
-                image.src = `${url.replace('metadata', 'download')}/${file.name}`;
-
-                imageRow.classList.add('d-flex', 'justify-content-between', 'align-items-center', 'mb-4', 'pe-5');
+                image.src = `${url.replace('metadata', 'download')}/${
+                  file.name
+                }`;
+
+                imageRow.classList.add(
+                  'd-flex',
+                  'justify-content-between',
+                  'align-items-center',
+                  'mb-4',
+                  'pe-5'
+                );
                 imageRow.append(image, placeButton);
 
                 imageContainer.appendChild(imageRow);
                 imageCount++;
               }
             });
-            responseText.innerHTML = imageCount ? `${imageCount} image(s) fetched successfully.` : 'No images found in the link provided...'
+            responseText.innerHTML = imageCount
+              ? `${imageCount} image(s) fetched successfully ${fetchedFrom.innerHTML}.`
```

Comment:
I think we may need the word "from" here? And if we are adding it to the same line, perhaps we don't need to make a new <p> ? Can you share a screenshot of how it looks? Thank you!

Reply:
Sure, just not caught that.

Pardon me, I don't understand what you meant by "we don't need to make a new?" could you clarify

Sure, I'll attach a screenshot right away.